model/ddl.go

```diff
@@ -107,12 +107,13 @@ func (h *HistoryInfo) Clean() {
 
 // Job is for a DDL operation.
 type Job struct {
-	ID       int64         `json:"id"`
-	Type     ActionType    `json:"type"`
-	SchemaID int64         `json:"schema_id"`
-	TableID  int64         `json:"table_id"`
-	State    JobState      `json:"state"`
-	Error    *terror.Error `json:"err"`
+	ID        int64         `json:"id"`
+	Type      ActionType    `json:"type"`
+	SchemaID  int64         `json:"schema_id"`
+	TableID   int64         `json:"table_id"`
+	TableName string        `json:"table_name"`
```

Comment:
Can we get the tablename by tableid?

Reply:
@shenli  `rename table` `tableid` has not been changed, so record the table name.

Reply:
Can we put this information to `Args`?